.jenkins/jenkins.sh

```diff
@@ -185,7 +185,7 @@ if [ ${python_env} == "virtualenv" ]; then
     if grep -q "parallel" <<< "${script}"; then
 	export MPIRUN_CALL="srun"
     fi
-    export pace_PATH="${envloc}/../"
+    export pace_PATH="${envloc}/../fv3gfs-physics/"
```

Comment:
This is no longer the path of PACE, can you either revert the path and append fv3gfs-physics later or rename the variable?

Also, if PATH is allcaps then PACE should be allcaps in this name, as much as I don't like allcaps PACE...